diff --git a/src/compiler/corePublic.ts b/src/compiler/corePublic.ts
index 532d2bb55..b45126951 100644
--- a/src/compiler/corePublic.ts
+++ b/src/compiler/corePublic.ts
@@ -1,6 +1,6 @@
 // WARNING: The script `configurePrerelease.ts` uses a regexp to parse out these values.
 // If changing the text in this section, be sure to test `configurePrerelease` too.
-export const versionMajorMinor = "5.8";
+export const versionMajorMinor = "0.1";
 // The following is baselined as a literal template type without intervention
 /** The version of the TypeScript compiler release */
 export const version: string = `${versionMajorMinor}.0-dev`;
